srcpkgs/python3-numpy/template

```diff
@@ -15,7 +16,22 @@ distfiles="https://github.com/numpy/numpy/archive/v${version}.tar.gz"
 checksum=43fe3bcdf27a2eb27196a1d140385aeece71c8b1a6eda18d2c6365239e50c60f
 alternatives="numpy:f2py:/usr/bin/f2py3"
 
+build_options="openblas"
+desc_option_openblas="Enable support for openblas accelerated linear algebra"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64*|i686*|aarch64*|armv[67]*|ppc64*)
+		# Prefer accelerated routines where available
+		build_options_default="openblas"
+		;;
+	*) ;;
+esac
+
 post_patch() {
+	if [ -n "$build_option_openblas" ] ; then
+		cp "${FILESDIR}/site.cfg.openblas" site.cfg
```

Comment:
This will not work properly for cross compilation. See what we do in `python3-scipy` for the right way to handle `site.cfg`.

Reply:
Changed it to generate the `site.cfg` file dynamically.